packages/shared/dao/reducer.ts

```diff
@@ -64,7 +64,7 @@ function realmProperties(realm: Realm, network: ETHEREUM_NETWORK | null): Partia
   // this condition exists until the BFF is finished and the "services" become interfaces
   if (realm.protocol === 'v1' || realm.protocol === 'v2') {
     return lighthouseBasedPartial(domain)
-  } else if (network === ETHEREUM_NETWORK.ROPSTEN) {
+  } else if (network === ETHEREUM_NETWORK.GOERLI) {
     return lighthouseBasedPartial('https://peer.decentraland.zone')
```

Comment:
We currently kept both networks: ropsten and goerli. `peer.decentraland.zone` and `peer-ue-2.decentraland.zone` stay on ropsten, and `peer-ap1.decentraland.zone` is on a separate DAO on goerli.
We did this so we can keep our current working environment while the new one gets to working state.